changelogs/unreleased/9570-remove-version-information-from-resource-attributes.yml

```
@@ -0,0 +1,6 @@
+description: Remove version information from resource attributes
```

```suggestion
description: Remove version information from resource attributes returned by the API
```

Perhaps this makes it a bit more informative? Considering that we include it in the changelog.